src/hotspot/share/runtime/vmThread.hpp

```diff
@@ -152,7 +111,8 @@ class VMThread: public NamedThread {
   void verify();
 
   // Performance measurement
-  static PerfCounter* perf_accumulated_vm_operation_time()               { return _perf_accumulated_vm_operation_time; }
+  static PerfCounter* perf_accumulated_vm_operation_time()
+    { return _perf_accumulated_vm_operation_time; }
```

Comment:
Why was this broken into two lines? If keeping that, then
the '{' belong on L114 at the end and not on L115.

Reply:
This line is very long and kept bugging me. Fixed '{'.